### ISSUE TITLE
[Bug Report] ... Sweep runner failure with T3K device

### ISSUE THREAD

**Issue by Aswinmcw** [tenstorrent/tt-metal#11847]
**Describe the bug**
While sweeping through all possible cases, some will fail but sweep infra fails to reset the device properly

It tells like,

```
SWEEPS: TEST TIMED OUT, Killing child process 3275722 and running tt-smi...
usage: tt-smi [-h] [-l] [-v] [-s] [-ls] [-f [filename]] [-g [GENERATE_RESET_JSON]] [-r 0,1 ... or config.json]
tt-smi: error: unrecognized arguments: -wr all
SWEEPS: TT-SMI Reset Failed with Exit Code: 2
```
tt-smi reset command is working with `-r` and not `-wr`
**Expected behavior**
Sweep should run without killing the process

**Screenshots**
<img width="1512" alt="Screenshot 2024-08-23 at 8 21 11 PM" src="https://github.com/user-attachments/assets/ebdd7bba-ec44-43ec-af51-80e1f60f10db">

cc. @SeanNijjar

**Comment by SeanNijjar** [comment]
Thanks @Aswinmcw.

Added note that this is for t3000 sweeps.

**Comment by jdesousa-TT** [comment]
@Aswinmcw Can you please confirm which machine this was run on?

**Comment by Aswinmcw** [comment]
> @Aswinmcw Can you please confirm which machine this was run on?

It is f10cs05 , T3k machine

**Comment by jdesousa-TT** [comment]
> @Aswinmcw Can you please confirm which machine this was run on? 

Nevermind, I can see in the screenshot which machine. This problem is a result of different tt-smi versions between machines.

**Comment by Aswinmcw** [comment]
> > @Aswinmcw Can you please confirm which machine this was run on?
> 
> Nevermind, I can see in the screenshot which machine. This problem is a result of different tt-smi versions between machines.

Yes so based on tt smi version, the reset command should work right?

**Comment by jdesousa-TT** [comment]
@Aswinmcw Merged PR #11901 which allows an override of the tt-smi command via environment variable. This will be the workaround for right now until we can standardize tt-smi versions across machines.

**Comment by Aswinmcw** [comment]
> @Aswinmcw Merged PR #11901 which allows an override of the tt-smi command via environment variable. This will be the workaround for right now until we can standardize tt-smi versions across machines.

Thanks!
